google/cloud/sql/connector/connector.py

```diff
@@ -36,12 +37,12 @@ def _get_loop() -> asyncio.AbstractEventLoop:
     return _loop
 
 
-def connect(instance_connection_string, driver: str, **kwargs):
+def connect(instance_conn_str, driver: str, **kwargs):
     """Prepares and returns a database connection object and starts a
     background thread to refresh the certificates and metadata.
 
-    :type instance_connection_string: str
-    :param instance_connection_string:
+    :type instance_conn_str: str
+    :param instance_conn_str:
         A string containing the GCP project name, region name, and instance
```

Comment:
If you're going to shorten this, update the docstring to include "instance connection string" and an explanation of the format, since that's a significant term in helping users find it in the console.